enterprise/wsproxy/proxy.go

```diff
@@ -0,0 +1,251 @@
+package wsproxy
```

Comment:
I think we should call this file `wsproxy.go` instead. Finding it amongst other files called `proxy.go` sounds annoying.